template-test/ci-run.sh

```diff
@@ -17,8 +17,8 @@ echo -e $GENERATOR_INPUT | RACK_ENV=development RAILS_ENV=development rails new
 
 # Run overcommit
 cd $APP_NAME
-gem install overcommit --no-document
-overcommit -r
+# gem install overcommit --no-document
+# overcommit -r
```

Comment:
I will remove this change before I commit. This change is here because Rubocop has new cops which emit warnings and still causes overcommit to fail. @joshmcarthur Do you think we need overcommit here in CI? CI already runs all our lints etc. - overcommit just gets triggered when we make git commits as part of some of the templates.

Reply:
@eoinkelly I think probably not. I think possibly the intention here was to make sure that CI would always run the checks that run on commit rather than doubling up the definitions of these elsewhere in the CI run